roles/node_exporter/molecule/default/Dockerfile.j2

```diff
@@ -0,0 +1,14 @@
+# Molecule managed
+
+{% if item.registry is defined %}
```

Comment:
Is it needed to add support for other distribution. (Yes we could, should we ?)

Reply:
This file is auto-generated by Molecule :smiley:

Indeed, this would allow to test against other platforms, by adding them in `molecule.yml`. However, this role currently doesn't support any platform but RedHat, and MetalK8s doesn't either, so not sure we should spend time on this...

Reply:
Could we restrict the platform supported in molecule.yml ? Hopefully, the generated code smaller, and not misleading.

Reply:
hmmm seems not

Reply:
There is already only one platform defined: https://github.com/scality/metal-k8s/pull/130/files#diff-20f42e7f8e26d9ec2ddfd316087ab50bR9

This `Dockerfile.j2` is managed by Molecule. Nobody is supposed to 'understand' it in a way :wink: